nvflare/fuel/f3/mpm.py

```diff
@@ -54,7 +54,7 @@ def get_aio_context(cls):
     @classmethod
     def logger(cls):
         if not cls._logger:
-            cls._logger = logging.getLogger(cls.name)
+            cls._logger = get_module_logger(cls.__module__, cls.name)
```

Comment:
would get_module_logger(__module__, __qualname__) work here?

Reply:
we want to use cls.name as it is an attribute that can be set with `set_name`